### ISSUE TITLE
Black screen

### ISSUE THREAD

**Issue by 1t3x** [sailfish-lavender/builds#15]
Hello,
First of all, thank you for the port update. I encountered a problem when flashing my phone. I wiped the partitions then flashed the 2 zips without any errors but once I restarted the phone, it is frozen on a black screen. I was wondering if there was a specific firmware version that needs to be installed on the phone?

Thank you for your time

**Comment by sumionochi** [comment]
right, same frozen/black screen was encountered by me too. although when ran version 3.3.0, it worked without any black screen.

**Comment by sumionochi** [comment]
> 
right, same frozen/black screen was encountered by me too. although when ran version 3.3.0, it worked without any black screen.

**Comment by FakeShell** [comment]
will look into it

**Comment by FakeShell** [comment]
make sure to downgrade to miui build  9 PKQ1.180904.001 V10.3.6.0.PFGMIXM or something close to that build first then do the installation
also after installing the zip package try this

chroot /data/.stowaways/sailfishos/ /bin/bash -c "export PATH=/bin:/sbin:/usr/bin:/usr/sbin && systemctl mask bluebinder && systemctl mask usb-moded"
then reboot and see if the system shows up

**Comment by Dolores95** [comment]
hello:), it also seems to me that the screen is black in versions 4. I have also noticed that the lighting sensors of the screen work even when the screen is black.

**Comment by FakeShell** [comment]
> hello:), it also seems to me that the screen is black in versions 4. I have also noticed that the lighting sensors of the screen work even when the screen is black.

please try the method i described above. i can get to ui without any issues which is very odd but others cannot.

**Comment by Dolores95** [comment]
> > hello:), it also seems to me that the screen is black in versions 4. I have also noticed that the lighting sensors of the screen work even when the screen is black.
> 
> please try the method i described above. i can get to ui without any issues which is very odd but others cannot.

Hello, thank you very much for the work. I have written this in twrp terminal. but I'm not an advanced user and I don't really know how to use it .chroot /data/.stowaways/sailfishos/ /bin/bash -c "export PATH=/bin:/sbin:/usr/bin:/usr/sbin && is for systemctl mask bluebinder && systemctl mask usb-moded".

I have also reinstalled it and first a black screen then it restarted on its own
after reboot hear charging sound and screen vibration but still black screen

**Comment by Dolores95** [comment]
> > hello:), it also seems to me that the screen is black in versions 4. I have also noticed that the lighting sensors of the screen work even when the screen is black.
> 
> please try the method i described above. i can get to ui without any issues which is very odd but others cannot.

Hello, thank you very much for the work. I have written this in twrp terminal. but I'm not an advanced user and I don't really know how to use it .chroot /data/.stowaways/sailfishos/ /bin/bash -c "export PATH=/bin:/sbin:/usr/bin:/usr/sbin && is for systemctl mask bluebinder && systemctl mask usb-moded".

I have also reinstalled it and first a black screen then it restarted on its own
after reboot hear charging sound and screen vibration,system seems to work, screenshot sound... ... but still black screen

**Comment by FakeShell** [comment]
> > > hello:), it also seems to me that the screen is black in versions 4. I have also noticed that the lighting sensors of the screen work even when the screen is black.
> > 
> > 
> > please try the method i described above. i can get to ui without any issues which is very odd but others cannot.
> 
> Hello, thank you very much for the work. I have written this in twrp terminal. but I'm not an advanced user and I don't really know how to use it .chroot /data/.stowaways/sailfishos/ /bin/bash -c "export PATH=/bin:/sbin:/usr/bin:/usr/sbin && is for systemctl mask bluebinder && systemctl mask usb-moded".
> 
> I have also reinstalled it and first a black screen then it restarted on its own after reboot hear charging sound and screen vibration,system seems to work, screenshot sound... ... but still black screen

Hi
thanks for the feedback and detailed description
I will make a new image and let you know

**Comment by smile4sky** [comment]
Hi
I have the same problem
maybe I did something wrong, do I have the right procedure?

1) installed orange fox R11.0
2) installed [fw_lavender_miui_LAVENDERGlobal_V10.3.6.0.PFGMIXM_8c74282d30_9.0.zip] ( 61.29 MB )
3) format data, reboot to recovery, then Wipe dalvik, data, cache, vendor, system
4) disabled force encryption in orange fox -> ota
5) installed this rom lineage-16.0-20200107-UNOFFICIAL-lavender.zip
6) reboot to recovery and again disable force encryption in orange fox -> ota
7) installed sailfishos-4.5.0.16-20230609-lavender.zip

and yes, after that, after the reboot, the black screen lights up, the light sensor works, the block button works, but the screen remains black
I entered this line in the recovery terminal, after it the same black screen:
chroot /data/.stowaways/sailfishos/ /bin/bash -c "export PATH=/bin:/sbin:/usr/bin:/usr/sbin && systemctl mask bluebinder && systemctl mask usb-moded"

my phone Redmi Note 7 3/32
has anyone found a way around this problem?

**Comment by FrAndroidd** [comment]
Same issue here. Also tried installing MIUI 10.3.6.0 and the firmware before doing everything and executing the command.
The screen is just black even though the light sensor works (it changes the screen's brightness).

The same thing happens with the latest firmware of MIUI 12.5, so I don't think the firmware has anything to do with this.

Any help? The old 3.3 build works, but SailfishOS 3 is quite old by now.

I can provide more information if needed.

**Comment by ghost** [comment]
Same issue here.
Followed all the steps, with MIUI V11.0.6.0.PFGMIXM and SailfishOS 4.5.0.16...
lavender 3/32, global version

**Comment by AlexanderShirokih** [comment]
Also facing with the same issue.
SailfishOS 4.5.0.16 does not works due to black screen on boot.
SailfishOS 3.3.0.16 works, but i cannot install or update anything due to certificate expiration error.

**Comment by KalWardinX** [comment]
was anyone able to resolve the blank screen issue.

**Comment by BokeeXD** [comment]
I have a Redmi Note 7 Global black 3/32 .Today I heard of SailfishOS and gave it a try with latest version, because LineageOS 16 did not want to flash under MIUI 12 firmware I resorted to MIUI 11.0.4.0 Global because that was the version right before the release of that LineageOS build. Originally I kept trying 4.5.0.16-20230609 but I kept getting the black display issue. Than I tried last pre prelease version which worked. I would use it if it was not for network issues while creating Jolla account.
